pkgs/applications/misc/cobang/default.nix

```diff
@@ -99,6 +99,6 @@ buildPythonApplication rec {
     license = licenses.gpl3Only;
     maintainers = [ ];
     mainProgram = "cobang";
-    platforms = [ "x86_64-linux" ];
+    platforms = [ "x86_64-linux" "aarch64-linux" ];
```

Comment:
```suggestion
    platforms = lib.platforms.linux;
```

Reply:
Yeah, just assume it works, since it's python

Reply:
I have updated the pull request to use `lib.platform.linux` instead

Reply:
> `lib.platform.linux`

![image](https://github.com/user-attachments/assets/617a73ae-f5b2-40ab-b492-50b5c3bf47c7)

Reply:
Thanks @superherointj, for noticing that I forgot a `s` in `platform` to should have been fixed